docs/quickstart.rst

```diff
@@ -183,7 +183,7 @@ Cancel a verification
     client.cancel_verification('00e6c3377e5348cdaf567e1417c707a5')
 
 Docs:
-`https://docs.nexmo.com/verify/api-reference/api-reference#control <https://docs.nexmo.com/verify/api-reference/api-reference#control?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
+`https://docs.vonage.com/verify/api-reference/api-reference#control <https://docs.vonage.com/verify/api-reference/api-reference#control?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
```

Comment:
This needs correct to point to the developer docs, this URL isn't valid. `docs.nexmo.com` is also just a redirect, so this should point to the current URL.